components/widgets/jenkins/index.js

```diff
@@ -0,0 +1,92 @@
+import { Component } from 'react'
+import fetch from 'isomorphic-unfetch'
+import styled from 'styled-components'
+import Widget from '../../widget'
+import Table, { Th, Td } from '../../table'
+import Badge from '../../badge'
+
+const JenkinsBadge = styled(Badge)`
+  background-color: ${props => {
+    switch (props.status) {
+      case 'FAILURE':
+        return props.theme.palette.errorColor
+      case 'UNSTABLE':
+        return props.theme.palette.warnColor
+      case 'SUCCESS':
+        return props.theme.palette.successColor
+      default:
+        return props.theme.palette.grey200
+    }
+  }}
+`
+
+export default class Jenkins extends Component {
+  static defaultProps = {
+    jobs: [],
+    title: 'Jenkins'
+  }
+
+  state = {
+    loading: false,
```

Comment:
At the moment every widget set the default loading state to `true`. If we need it differently in the future, we will change it later.